Managers/SceneMan.cpp

```diff
@@ -950,58 +948,42 @@ bool SceneMan::TryPenetrate(int posX,
             int testMaterialID = g_MaterialAir;
             MOPixel *pixelMO = 0;
             Color spawnColor;
-            float sprayMag = velocity.GetLargest() * sprayScale;
-            Vector sprayVel;
+			float sprayMag = std::sqrt(velocity.GetMagnitude() * sprayScale);
```

Comment:
why sqrt?

Reply:
No particular geometrical reason, just to exponentially diminish the velocity of the dislodged pixels

Reply:
Any reason why the * sprayScale is done before the sqrt and not after? If we do it after, we can use a std::cbrt() of the sqr magnitude instead, which may be more performant than 2 square roots.

Reply:
With the sprayscale I was just adhering to the way it's used higher up in the block, but I suppose it ultimately doesn't matter